recipes-bsp/tegra-binaries/tegra-nvphs-base_34.1.0.bb

```diff
@@ -15,6 +15,7 @@ SRC_URI[init.sha256sum] = "${INITSUM}"
 NVPHSD_MACHINE_CONF = "nvphsd.conf"
 NVPHSD_MACHINE_CONF:tegra186 = "nvphsd.conf.t186"
 NVPHSD_MACHINE_CONF:tegra194 = "nvphsd.conf.t194"
+NVPHSD_MACHINE_CONF:tegra234 = "nvphsd_common.conf"
```

Comment:
Looks like nvphsd doesn't need this config renaming stuff any more, and we should just install both of the config files into `${D}${sysconfdir}` as-is, for all platforms.